sonar-vbnet-plugin/src/main/java/org/sonar/plugins/vbnet/VbNetPlugin.java

```diff
@@ -62,6 +61,7 @@ public void define(Context context) {
       RoslynProfileExporter.class);
 
     context.addExtensions(new VbNetPropertyDefinitions(context.getRuntime()).create());
+    context.addExtension(new VbNetSonarWayProfile(context.getRuntime()));
```

Comment:
There is no default ctor on the class now and we need to register the instance through a different method